policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/exception/AuditableException.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.palisade.service.policy.exception;
+
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+
+import uk.gov.gchq.palisade.Generated;
+import uk.gov.gchq.palisade.service.policy.model.PolicyRequest;
+
+/**
+ * Capture a thrown exception and a kafka consumer record, wrapping them in a message to be caught by the supervisor.
+ * This allows exceptions to be audited along with their original request.
+ * These exceptions are elevated to {@link RuntimeException}s.
+ */
+public class AuditableException extends RuntimeException {
```

Comment:
Since we're not using this left, maybe best to not include?

Reply:
removed